devel/lab/Portfile

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        zaquestion lab 0.14.0 v
+categories          devel
+platforms           darwin
+license             The Unlicense
+maintainers         {cwrinn @cwrinn}
+
+description         git-wrapper for GitLab specific functionality
+long_description    \
+    Lab wraps Git or Hub, making it simple to clone, fork, and interact with \
+    repositories on GitLab, including seamless workflows for creating merge \
+    requests, issues and snippets.
+
+homepage            https://zaquestion.github.io/lab
+
+checksums           rmd160  ce839f740ce45ccfaaafac19626bb1827816cce5 \
+                    sha256  e423dd0b524ad16836306566232d5bc7ccffd67d2b98b39d5493e0d529597857 \
+                    size    73519
+
+worksrcdir          src/github.com/${github.author}/${name}
```

Comment:
The portgroup doesn't handle this bit already? (I confess I haven't checked.)

Reply:
So, **confession**: I referenced what devel/hub did, since it was the closest similar port. If that's unnecessary, more than happy to learn to do it properly 👍

Reply:
Switched to PortGroup golang